azurerm/internal/services/servicebus/resource_arm_servicebus_queue.go

```diff
@@ -73,25 +81,50 @@ func resourceArmServiceBusQueue() *schema.Resource {
 				ValidateFunc: validate.ISO8601Duration,
 			},
 
+			"enable_batched_operations": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Default:  true,
+			},
+
 			"enable_express": {
 				Type:     schema.TypeBool,
-				Default:  false,
 				Optional: true,
+				Default:  false,
 			},
 
 			"enable_partitioning": {
 				Type:     schema.TypeBool,
-				Default:  false,
 				Optional: true,
+				Default:  false,
 				ForceNew: true,
 			},
 
+			"forward_dead_lettered_messages_to": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				ValidateFunc: azure.ValidateServiceBusQueueName(),
+			},
+
+			"forward_to": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				ValidateFunc: azure.ValidateServiceBusQueueName(),
+			},
+
 			"lock_duration": {
 				Type:     schema.TypeString,
 				Optional: true,
 				Computed: true,
 			},
 
+			"max_delivery_count": {
+				Type:         schema.TypeInt,
+				Optional:     true,
+				Default:      10,
+				ValidateFunc: validation.IntAtLeast(1),
+			},
+
 			"max_size_in_megabytes": {
```

Comment:
I thought about adding a small validation function there, something like

```go
"max_size_in_megabytes": {
	Type:         schema.TypeInt,
	Optional:     true,
	Computed:     true,
	ValidateFunc: validation.IntInSlice([]int{1024, 2048, 3072, 4096, 5120}),
},
```

Reply:
If it can only be 1-5 megabytes this is a good idea! however i may review and merge this to get it into todays release so another PR would be welcome :)